quartz/plugins/transformers/ofm.ts

```diff
@@ -159,6 +163,16 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
         })
       }
 
+      if (opts.parseArrows) {
```

Comment:
let's implement this in markdown only in text nodes by pushing to the replacements array (this shouldn't transform codeblocks or math for example)

Reply:
implemented. PTAL.